cfgrammar/src/lib/newlinecache.rs

```diff
@@ -3,16 +3,14 @@ pub struct NewlineCache {
     trailing_bytes: usize,
 }
 
-impl Default for NewlineCache {
-    fn default() -> NewlineCache {
-        NewlineCache {
+impl NewlineCache {
+    pub fn new() -> Self {
```

Comment:
:+1:  :100: I only did this so clippy wouldn't trigger on it,
I only now realize that `#![allow(clippy::new_without_default)]` is enabled.